### PR TITLE
depedencies: update minor and patch

### PR DESCRIPTION
**Pull request by desimone**
## Summary
Updates all our dependencies to the latest patch and minor versions. Wondering if it makes sense to use something like #403 to automate this. But honestly, it's a pretty straightforward process with gomods. 

```
go list -u -f '{{if (and (not (or .Main .Indirect)) .Update)}}{{.Path}}: {{.Version}} -> {{.Update.Version}}{{end}}' -m all 2> /dev/null
```

```
contrib.go.opencensus.io/exporter/jaeger: v0.1.0 -> v0.2.0
github.com/prometheus/client_golang: v0.9.3 -> v1.2.1
github.com/rs/zerolog: v1.16.0 -> v1.17.2
github.com/spf13/viper: v1.4.0 -> v1.5.0
go.opencensus.io: v0.22.0 -> v0.22.2
golang.org/x/crypto: v0.0.0-20191106202628-ed6320f186d4 -> v0.0.0-20191122220453-ac88ee75c92c
golang.org/x/net: v0.0.0-20191028085509-fe3aa8a45271 -> v0.0.0-20191125084936-ffdde1057850
golang.org/x/oauth2: v0.0.0-20190604053449-0f29369cfe45 -> v0.0.0-20191122200657-5d9234df094c
google.golang.org/api: v0.13.0 -> v0.14.0
google.golang.org/grpc: v1.24.0 -> v1.25.1
gopkg.in/yaml.v2: v2.2.3 -> v2.2.7
```

**Checklist**:
- [x] ready for review

### CODE REVIEW

**Comment by travisgroth** [comment]
👍 
I really like the idea of the bot, especially since it looks to be proactive.

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -12,8 +12,8 @@ categories:
     label: removed
   - title: Documentation
     label: docs
-  - title: Depedency
-    label: depedency
+  - title: Dependency
```
Comment (travisgroth) [inline .github/release-drafter.yml]: 😆

Comment (desimone) [reply .github/release-drafter.yml]: 🙃

**Comment by desimone** [comment]
> 👍
> I really like the idea of the bot, especially since it looks to be proactive.

I've updated this PR with `/renovate.json`, so we'll see how that goes 👍

**Pull request merged**